game/scripts/npc/abilities/talents/gyrocopter_talent3.txt

```diff
@@ -0,0 +1,26 @@
+"DOTAAbilities"
+{
+  //=================================================================================================================
+  // Bonus Rocket Barrage Damage
+  //=================================================================================================================
+  "special_bonus_unique_gyrocopter_3"
+  {
+    // General
+    //-------------------------------------------------------------------------------------------------------------
+    "ID"					""														// unique ID number for this ability.  Do not change this once established or it will invalidate collected stats.
+    "AbilityType"					"DOTA_ABILITY_TYPE_ATTRIBUTES"
+    "AbilityBehavior"				"DOTA_ABILITY_BEHAVIOR_PASSIVE"
+
+    // Special
+    //-------------------------------------------------------------------------------------------------------------
+    "AbilitySpecial"
+    {
+      "01"
+      {
+        "var_type"					"FIELD_INTEGER"
+        "value"				"32" //OAA
```

Comment:
I wouldn't play with this talent. Even a slight increase is really good because of the nature of the spell. Tone it down or revert.

Reply:
I would put this at 24.